dev/bots/utils.dart

```diff
@@ -260,3 +300,297 @@ Future<bool> _isPortAvailable(int port) async {
 String locationInFile(ResolvedUnitResult unit, AstNode node, String workingDirectory) {
   return '${path.relative(path.relative(unit.path, from: workingDirectory))}:${unit.lineInfo.getLocation(node.offset).lineNumber}';
 }
+
+String? _shuffleSeed;
+
+String get shuffleSeed {
```

Comment:
Should this not have the comment from https://github.com/flutter/flutter/pull/146592/files#diff-1d430014792a8013d6ad4c147b8e7e1bf3f99574b7af317a321f1ba752854503L187-L189 ?

Reply:
Updated to retain the original comment